ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionIndexing.scala

```diff
@@ -77,6 +77,15 @@ object TransactionIndexing {
     private val disclosure = Map.newBuilder[NodeId, Set[Party]]
     private val visibility = Vector.newBuilder[(ContractId, Set[Party])]
 
+    init()
```

Comment:
What do you think about inlining the code in `init` to the "class body". It seems like this short line is easier to overlook as part of the constructor than the for comprehension itself.

Reply:
Yes, it probably looks simpler and easier to catch now.